scoper.inc.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * PHP-Scoper configuration file.
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+use Isolated\Symfony\Component\Finder\Finder;
+
+// Google API services to include classes for.
+$google_services = implode(
+	'|',
+	array_map(
+		function( $service ) {
+			return preg_quote( $service, '#' );
+		},
+		array(
+			'Analytics',
+			'AnalyticsReporting',
+			'AdSense',
+			'Pagespeedonline',
+			'PeopleService',
+			'SiteVerification',
+			'TagManager',
+			'Webmasters',
+		)
+	)
+);
+
+return array(
+	'prefix'                     => 'Google\\Site_Kit_Dependencies',
+	'finders'                    => array(
+
+		// General dependencies, except Google API services.
+		Finder::create()
+			->files()
+			->ignoreVCS( true )
+			->notName( '/LICENSE|.*\\.md|.*\\.dist|Makefile|composer\\.json|composer\\.lock/' )
+			->exclude(
+				array(
+					'doc',
+					'test',
+					'test_old',
+					'tests',
+					'Tests',
+					'vendor-bin',
+				)
+			)
+			->path( '#^firebase/#' )
+			->path( '#^google/apiclient/#' )
+			->path( '#^google/auth/#' )
+			->path( '#^guzzlehttp/#' )
+			->path( '#^monolog/#' )
+			->path( '#^psr/#' )
+			->path( '#^ralouphie/#' )
+			->path( '#^react/#' )
+			->in( 'vendor' ),
+
+		// Google API service infrastructure classes.
+		Finder::create()
+			->files()
+			->ignoreVCS( true )
+			->notName( '/LICENSE|.*\\.md|.*\\.dist|Makefile|composer\\.json|composer\\.lock/' )
```

Comment:
```suggestion
			->notName( '/LICENSE|.*\\.md|.*\\.dist|Makefile|composer\\.(json|lock)/' )
```

Reply:
Looks like this change should still be made.